services/src/main/java/org/keycloak/services/resources/admin/GroupsResource.java

```diff
@@ -40,10 +42,9 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import java.net.URI;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Please, don't use wildcards. Import specific types instead. This change is not really necessary. The new types you are importing are fine.